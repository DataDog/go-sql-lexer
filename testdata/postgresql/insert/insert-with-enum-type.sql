INSERT INTO shipments (status) VALUES ('delivered'::shipment_status);
