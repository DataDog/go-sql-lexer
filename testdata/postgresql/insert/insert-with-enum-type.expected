[
    {
        "expected": "INSERT INTO shipments ( status ) VALUES ( ? :: shipment_status )",
        "statement_metadata": {
            "size": 15,
            "tables": [
                "shipments"
            ],
            "commands": [
                "INSERT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]